content/events/2025-09-22-graz-workshop/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: From data to discovery - Galaxy workshop at University of Graz
+date: '2025-09-22'
+days: 5
+tease: "This course introduces scientists to the data analysis platform Galaxy. The course is an intermediate course; there is no requirement of any programming skills."
+continent: EU
+location:
+  name: "University of Graz, Austria"
+gtn: true
+contact: "Amirhossein Nilchi, Pavan Videm, Sebastian Preissl"
+tags: [workshop, training]
+supporters:
+- University of Graz
+subsites: [all]
+---
+
+# General information
+
+- **Date**: September 24-26, 2025
+- **Format**: on-site
+- **Location**: university of Graz, Graz, Austira
+- **Time**: 9:15am-5pm CEST
+- **Cost**: Free
+- **Contact**: Amirhossein Nilchi ([nilchia@informatik.uni-freiburg.de](mailto:nilchia@informatik.uni-freiburg.de))
+
+# Pedagogical information
+
+**Target Audience**: Scientist with no or little Galaxy experience who want to analyse high-throughput data.
+
+**Learning Objectives:**
+* To get introduced to Galaxy as data analysis platform and the GTN training material
+* To learn how to use use tools and databases in Galaxy
+* To be able to run different tools and workflows
+
+**Training material:**
+For the hands-on course, we will use the [training material](https://training.galaxyproject.org) from the Galaxy Training Network.
+
```

Comment:
can you please include some specifics about the content, like intro, rna-seq, proteo and metabolomics. Otherwise, make a table of schedule.